src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -134,7 +134,15 @@ public Loading(LoadingStyle loadingStyle) : base(loadingStyle)
         /// <since_tizen> 6 </since_tizen>
         public string[] ImageArray
         {
-            get => (GetValue(ImageListProperty) as List<string>).ToArray();
+            get
+            {
+                string[] ret = null;
+                if (GetValue(ImageListProperty) is List<string>)
+                {
+                    ret = (GetValue(ImageListProperty) as List<string>).ToArray();
+                }
+                return ret;
+            }
```

Comment:
It is equivalent to your code but more readable and it check null so it can pass on Svase
```suggestion
            get => (GetValue(ImageListProperty) as List<string>)?.ToArray() ?? null;
```